package.json

```diff
@@ -5,8 +5,9 @@
   "main": "index.js",
   "scripts": {
     "start": "node build/index.js",
+    "pretest": "npm run migrate:undo && npm run migrate && npm run seed",
     "test": "cross-env DEBUG=dev jest --forceExit --verbose --maxWorkers=1 --coverage",
-    "coverage": "jest --forceExit --coverage && cat ./coverage/lcov.info | coveralls",
+    "coverage": "npm run pretest && jest --forceExit --coverage && cat ./coverage/lcov.info | coveralls",
```

Comment:
Why is npm run pretest here

Reply:
Because coverage was failing

Reply:
It was still seeing the data when it runs, the scripts drops and create for coverage to work with fresh data

Reply:
Ok